tests/pynfs/run.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run tests
+# Maintainer: Yong Sun <yosun@suse.com>
+package run;
+
+use 5.018;
+use strict;
+use warnings;
+use base 'opensusebasetest';
+use File::Basename;
+use testapi;
+use utils;
+use power_action_utils 'power_action';
+
+sub server_test_all {
+    my $folder = shift;
+    assert_script_run("cd ./$folder");
+    script_run('./testserver.py --outfile ${logf:-log.txt} --maketree ${serv:-localhost}:${export:-/exportdir} all', 3600);
+}
+
+
+sub run {
+    my $self = shift;
+    select_console('root-console');
+
+    my $test_scope = get_var('PYNFS');
+    server_test_all($test_scope);
```

Comment:
I'd move `get_var('PYNFS')` to `server_test_all()` and use `get_required_var()` or have a default fallback when not defined (default might not be needed as tests aren't going to be used without `PYNFS`, but `get_required_var()` is kind of documentation what is needed.

Reply:
Also maybe document `PYNFS` with perldoc in `tests/pynfs/run.pm`?

Reply:
Thank you for advice! I modified the code, and write document in run.pm.